SuperSquareBros.cpp

```diff
@@ -1782,6 +1763,7 @@ class Entity {
                         x = foreground[i].x + SPRITE_SIZE - 1;
                     }
                     xVel = 0;
+                    break;
```

Comment:
I went a step further and skipped the entire loop if `xVel` was 0. https://github.com/ThePythonator/Super-Square-Bros/commit/967e332b93a14afa08842189d6ab832bcbf5d050#diff-ab6f4f9204ca41ac0f2c5132e6d8507701d0eba631f4db894a75bf99b6c65ad7R1755